_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -63,13 +58,12 @@ _There are no major accessibility support issues known for this rule._
 
 [Visual reference words][] that can be interpreted with the non-sensory meaning include, in English, expressions like "right after this" where "right" is a [visual reference word][] used with the meaning "immediately"; or words like "below" that is often used with the meaning "further in reading order".
 
-The rule doesn't require the non-visual characteristic description to be included in the accessibility tree. If the alternatives are not included in the accessibility tree, only [Success Criteria 1.3.1 Info and Relationships][sc131] would fail instead of both [Success Criterion 1.3.3 Sensory Characteristics][sc133] and [Success Criteria 1.3.1 Info and Relationships][sc131]. Hence, the rule passes in these cases as it is not a failure of all accessibility requirements.
+The rule doesn't require the non-visual characteristic description to be included in the accessibility tree. If the alternatives are not included in the accessibility tree, only [Success Criteria 1.3.1 Info and Relationships][sc131] would fail instead of [Success Criterion 1.3.3 Sensory Characteristics][sc133]. Hence, the rule passes in these cases as it is not a failure of the accessibility requirements of this rule.
```

Comment:
My only confusion is how passed example 12 (example with description not included in the accessibility tree) was passing with 1.3.1 is/was passing to begin with. We typically don't like examples that don't pass all criteria unless they are extensions of each other (e.g., AA and AAA for color contrast). If someone could clear that up would be great, but otherwise I think this is good to go.